docs/source/docs/command-reference/context_free/prune.md

```diff
@@ -0,0 +1,97 @@
+    If true, this operation will be saved in the project build tree, allowing to reproduce the resulting dataset later.
+    Applicable only to main project targets (i.e. data sources and the `project` target, but not intermediate stages). Enabled by default.
+
+Examples:
+- Prune dataset through clustering random with image hash into ratio 80
+```console
+datum prune source1 -m cluster_random -h image -r 80
+```
+
+### Built-in prune methods
+- [`random`](#random) - Select randomly among dataset
+- [`cluster_random`](#cluster_random) - Select randomly among clusters
+- [`centroid`](#centroid) - Select center of each cluster
+- [`query_clust`](#query_clust) - Set init with label of cluster
+- [`entropy`](#entropy) - Select dataset based label entropy among clusters
+- [`ndr`](#ndr) - Removes duplicated images from dataset
+
+#### `random`
+Randomly select items from the dataset. The items are chosen from the entire dataset using the most common random method.
+```console
+datum prune -m random -r 0.8 -p </path/to/project/>
+```
+
+#### `cluster_random`
+Select items randomly among each clusters. Cluster the entire dataset using K-means based on the number of labels, and select items from each cluster according to the desired ratio.
+```console
+datum prune -m cluster_random -r 0.8 -p </path/to/project/>
+```
+![cluster_random](../../../../images/cluster_random.png)
+
+#### `centroid`
+Clustering the entire dataset, set the number of desired data samples as the number of clusters. To perform clustering, a desired number of data points is selected as centroids from the entire dataset, based on the desired proportion. Then, the centers of each cluster are chosen.
+
+```console
+datum prune -m centroid -r 0.8 -p </path/to/project/>
+```
+![centroid](../../../../images/centroid.png)
+
+#### `query_clust`
+When clustering the entire dataset, set the representative query for each label as the center of the cluster. The representative query is calculated through image or text hash, and one representative query is set for each label. It supports the approach of randomly selecting one item per label and choosing it as the representative query. In the generated clusters, random selection of items is performed according to the desired ratio.
+```console
+datum prune -m query_clust -h img -r 0.8 -p </path/to/project/>
+```
+
+```console
+datum prune -m query_clust -h txt -r 0.8 -p </path/to/project/>
+```
+![query_clust](../../../../images/query_clust.png)
+
+#### `entropy`
+After clustering the entire dataset, items are selected within each cluster based on the desired ratio, considering the entropy of labels.
+```console
+datum prune -m entropy -r 0.8 -p </path/to/project/>
+```
+![entropy](../../../../images/entropy.png)
+
+#### `ndr`
+Removes near-duplicated images in subset. You could check detail for this method in [ndr](./transform.md#ndr).
+We set `num_cut` to a size that the ratio that will be left in the entire dataset.
```

Comment:
What is `num_cut`? It seems not configurable by users.

Reply:
This parameter is used in `ndr`. So I tried to explain `num_cut` is set as a size that the ratio will be left in the entire dataset.

Reply:
`num_cut` seems the number of items remaining after NDR.

https://github.com/openvinotoolkit/datumaro/blob/0ec221609f28584bef656a67aa55cc73498459fc/src/datumaro/plugins/ndr.py#L37-L38

However, it seems that our readers need not know about this. Because they already know they can use 'ratio' to scale the already pruned dataset. It just seems adding some confusing.

Reply:
I see. I will remove this line. Thank you for suggestion.